postgres-appliance/bootstrap/clone_with_wale.py

```diff
@@ -82,11 +85,11 @@ def run_clone_from_s3(options):
 def main():
     options = read_configuration()
     try:
-        return run_clone_from_s3(options)
+        return run_clone_from_s3(options) and call_maybe_pg_upgrade()
```

Comment:
seeing this twice, any way this can be merged?

and the exception log line could then also hin on "upgrade failed".

Reply:
There is no "twice".

Error messaging indeed could be improved.

Reply:
Could in general need some more logging? The upgrade Python code looks very empty.